RecoLocalTracker/Configuration/python/RecoLocalTracker_cff.py

```diff
@@ -9,11 +9,11 @@
 from RecoLocalTracker.SiStripRecHitConverter.StripCPEfromTrackAngle_cfi import *
 from RecoLocalTracker.SiStripZeroSuppression.SiStripZeroSuppression_cfi import *
 from RecoLocalTracker.SiStripClusterizer.SiStripClusterizer_cfi import *
-from RecoLocalTracker.SiPixelClusterizer.SiPixelClusterizerPreSplitting_cfi import *
+from RecoLocalTracker.SiPixelClusterizer.siPixelClustersPreSplitting_cff import *
```

Comment:
No, it is part of the topic of this PR. The change is needed because `siPixelClustersPreSplitting` is no longer produced by a single producer, but by multiple producers.